app/javascript/components/Study.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+
+const descriptionCharLimit = 750
+
+export function formatDescription(rawDescription) {
+
+  let textDescription = stripTags(rawDescription)
+  let suffixString = ''
+  if (textDescription.length > descriptionCharLimit) {
+    suffixString = '... (continued)'
+  }
```

Comment:
This is more readable to me than its "const-and-ternary" equivalent.

Reply:
yeah, I tend to overuse ternaries, so I'm glad I have you as a counterweight.  Ternaries are really only good when the condition and assignments are really simple and short